swf/src/write.rs

```diff
@@ -991,12 +991,7 @@ impl<W: Write> Writer<W> {
                 } else {
                     writer.write_u16(0)?;
                 }
-                let mut flags = action.conditions.bits();
-                if action.conditions.contains(ButtonActionCondition::KEY_PRESS) {
```

Comment:
Just curious - the writing bug was this condition here, right?

Reply:
We pretended that the key press condition was `1<<9`, so when this bit wrote the `key<<9` in addition to the condition bit flags, it'd effectively write `(key | 1) << 9`

Reply:
Ah, okay, right, thanks.